tests/test_train.py

```diff
@@ -164,6 +166,43 @@ def test_train_nlu_wrong_format_error_message(
     assert "Please verify the data format" in captured.out
 
 
+def test_train_nlu_with_responses_or_domain_warns(
```

Comment:
as we are training two models here and model trainings are evil in terms of CI runtime: Let's use a minimal config (only featurizer, response selector with 1 epoch) to speed this up.